client/blocks/color-scheme-picker/docs/example.jsx

```diff
@@ -0,0 +1,43 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { PureComponent } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Notice from 'components/notice';
+import ColorSchemePicker from 'blocks/color-scheme-picker';
+
+class ColorSchemePickerExample extends PureComponent {
+	static displayName = 'ColorSchemePicker';
+
+	state = {
+		selectedColorScheme: null,
+	};
+
+	handleColorSchemeSelection = colorScheme => {
+		this.setState( { selectedColorScheme: colorScheme } );
+	};
+
+	render() {
+		return (
+			<Card>
+				<ColorSchemePicker
+					temporarySelection={ true }
+					onSelection={ this.handleColorSchemeSelection }
+				/>
+
+				{ this.state.selectedColorScheme &&
+					<Notice
+						text={ `Selected color scheme: ${ this.state.selectedColorScheme }` }
```

Comment:
Good point, I think I'd just forgotten I was looking at an example for a moment! Since it is an example this looks perfect - thanks for correcting me ;)